### PR TITLE
Log if external source map is not found with loadMaps: true

### PR DESCRIPTION
**Pull request by ogvolkov**
Let's give a user at least some clue as to what's wrong.

### CODE REVIEW

File: src/init/index.internals.js [hunk 1]
```diff
@@ -112,7 +114,9 @@ module.exports = function(options, file, fileContent) {
 
     try {
       sources.map = JSON.parse(stripBom(fs.readFileSync(mapFile, 'utf8')));
-    } catch (e) {} //should we really swallow this error?
+    } catch (e) {
+      debug(function() { return 'warn: external source map not found or invalid: ' + mapFile; });
```
Comment (nmccready) [inline src/init/index.internals.js]: Should we log anything from `e`? Should we log the stack trace? Otherwise LGTM

Comment (ogvolkov) [reply src/init/index.internals.js]: Hmmm, my use case was a missing file, and the similar code in _fixSources couldn’t care less about e, but now when I look at JSON.parse and stripBom, it seems beneficial to include something out of the exception indeed. How would you propose to stringify it?

As for the stack, I think that would be overkill as the logger already has a location and it’s more or less narrow.

**Comment by ogvolkov** [comment]
I need some help with the coverage. If I understand it correctly, the tool is too clever by half and wants me to have a branch to test an exception object without a message property.

First of all, is it possible to get the same coverage reports locally?

On the issue itself, I do not see a way to simulate such an exception (both file not found and invalid json have a message like they should). What I could do is to create exceptionToLog method in utils, use it in index.internals.js, and create test/utils.js containing tests for the two branches of that method.

Is it a recommended way, or maybe somebody can suggest something less impactful?

**Comment by ogvolkov** [comment]
Okay, I've fixed the coverage.
@nmccready can you please look if the changes make sense?

**Comment by nmccready** [comment]
LGTM

**Pull request merged**